### ISSUE TITLE
Required Inputs on Create a New Book page lack required attribute & appropriate label

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#2435]
The webbook address and book title inputs on the Create a New Book page are required but do not have a required attribute or label that states that the input needs to be filled. In addition the error messages are not associated with the input using an aria-describedby attribute. 
![Screenshot from 2021-10-12 15-44-07](https://user-images.githubusercontent.com/13485451/137038841-a1a97f0c-ebfb-41e8-899c-644e1fe5ccc4.png)

https://github.com/pressbooks/pressbooks/blob/dev/inc/registration/namespace.php

Finally, the Webbook Address input on the Create a New Book page has a label. However this label does not include the URL content that is included in the address by default. As a result, users may type in the entire URL resulting in an invalid value
![Screenshot from 2021-10-12 15-41-22](https://user-images.githubusercontent.com/13485451/137038599-de787a93-24e0-4941-998b-af54b1b44f0b.png)

**Comment by SteelWagstaff** [comment]
Filed upstream with WordPress: https://core.trac.wordpress.org/ticket/54344#ticket

**Comment by greatislander** [comment]
Will be resolved in forthcoming WordPress 6.1 release, see: https://core.trac.wordpress.org/ticket/54344
